### PR TITLE
update docs

### PR DESCRIPTION
**Pull request by noronhaa**
### JIRA link (if applicable) ###

n/a

### Change description ###
Made some changes/improvements to setup docs based on personal experience

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,6 +124,15 @@ Parameters:
 - `role`: Name of the role, e.g: `caseworker-divorce`.
 - `classification`: Optional. One of `PUBLIC`, `PRIVATE` or `RESTRICTED`. Defaults to `PUBLIC`.
 
+#### Note:
+If you get `{"timestamp":"2019-07-16T09:11:57.403+0000","status":403,"error":"Forbidden","message":"Access Denied","path":"/api/user-role"}` error you may need to install the `jq` library like so:
```
Comment (MSancaktutar) [inline README.md]: There seems to be a confusiopn here. That http 403 is not about jq installation.

File: README.md [hunk 2]
```diff
@@ -118,6 +124,15 @@ Parameters:
...
+```bash
+brew update;
+brew install jq;
+```
+
+If you are using Windows you should buy a Mac at this point :)
```
Comment (MSancaktutar) [inline README.md]: There are windows users working with a win release of jq.
jq is already listed in the required tools above. And, installation link is already provided.
So, this part may not be so necessary.

File: README.md (hunk 1 above)
Comment (noronhaa) [reply README.md]: hmm that is strange, do we know why the 403 happens? I had a 403, I installed the jq library and ran the same command and it worked after that

File: README.md (hunk 2 above)
Comment (noronhaa) [reply README.md]: yes noted, will remove this line

File: README.md [hunk 3]
```diff
@@ -118,6 +124,12 @@ Parameters:
 - `role`: Name of the role, e.g: `caseworker-divorce`.
 - `classification`: Optional. One of `PUBLIC`, `PRIVATE` or `RESTRICTED`. Defaults to `PUBLIC`.
 
+#### Note:
```
Comment (MSancaktutar) [inline README.md]: could we please remove this note section? jq is already listed in the required tools list and a link to its own installation page is provided.

Comment (noronhaa) [reply README.md]: done

**Pull request merged**